frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -57,6 +58,7 @@ const validateVm: Validator = (field, options) => {
         : VIRTUAL_MACHINE_EXISTS,
       subject: getFieldTitle(iGetFieldKey(field)),
     },
+    immutableListToJS(iGet(field, 'validations')) as UIValidation[],
```

Comment:
probably better to enhance the object arg than positional arguments

Reply:
Done!